<launch>

  <include file="$(find anymal_sim_cdt)/launch/sim.launch">
    <arg name="world"                    value="cdt_challenge"/>

    <!-- origin - near landing: 0.0, 0.0, 0.8, 0.0, 0.0, 0.0 -->

    <!-- ori outside of building: -6.5, 0.0, 0.0, 0.0, 0.0, 0.0 -->
    <!-- ori bottom of entrance staircase: -4.5, 0.0, 0.0, 0.0, 0.0, 0.0 -->
    <!-- ori top of entrance staircase: -2.5, 0.0, 0.8, 0.0, 0.0, 0.0 -->
    <!-- ori bottom of main staircase: 3.05, 3.0, 0.8, 0.0, 0.0, 0.0 -->
    <arg name="x"                                       default="0.0"/>
    <arg name="y"                                       default="0.0"/>
    <arg name="z"                                       default="0.8"/>
    <arg name="yaw"                                     default="0.0"/>

  </include>
  
  <rosparam param="/cdt_challange/goal_x">14.5</rosparam>
  <rosparam param="/cdt_challange/goal_y">4.0</rosparam>

  <!-- <node pkg="anymal_sim_cdt" type="goal_placer_node" name="goal_placer_node" output="screen"></node> -->

  <!-- <rosparam param="/cdt_challange/goals_x">[0.0, 3.0, 7.0, 14.5]</rosparam> -->
  <!-- <rosparam param="/cdt_challange/goals_y">[0.0, 2.0, 4.0, 4.0]</rosparam> -->

</launch>
